src/components/DetailCarousel/DetailCarousel.js

```diff
@@ -6,13 +7,14 @@ import { faChevronLeft } from '@fortawesome/free-solid-svg-icons';
 import { faChevronRight } from '@fortawesome/free-solid-svg-icons';
 import { faUserCircle } from '@fortawesome/free-solid-svg-icons';
 import { faThumbsUp } from '@fortawesome/free-regular-svg-icons';
-import { API } from '../../API/api';
+import { API_ENDPOINT } from '../../api';
 
 const SLIDERWIDTH = 732;
 
 function DetailCarousel(props) {
```

Comment:
이렇게 쓰시는 것 보다는 props 받아 올 때 바로 구조분해 할당 해주시면 좋을 것 같습니다.

```suggestion
function DetailCarousel({ pathParameterId }) {
```